xscen/extract.py

```diff
@@ -912,7 +913,14 @@ def subset_warming_level(
         end_yr = int(yr + window / 2)
 
     if np.isnan(start_yr):
-        logger.info(f"Global warming level of +{wl}C is never reached for {id_ds}.")
+        logger.info(
+            f"Global warming level of +{wl}C is not reached by 2100 for {id_ds}."
```

Comment:
technically, the user could provide a cvs that goes past 2100... How about f"...is not reached by the latest year in {tas_csv}." ?